x-pack/solutions/security/plugins/security_solution/public/attack_discovery/pages/results/attack_discovery_panel/actionable_summary/index.tsx

```diff
@@ -27,6 +29,13 @@ const ActionableSummaryComponent: React.FC<Props> = ({
   replacements,
   showAnonymized = false,
 }) => {
+  const {
+    application: { capabilities },
+  } = useKibana().services;
+  // TODO We shouldn't have to check capabilities here, this should be done at a much higher level.
+  //  https://github.com/elastic/kibana/issues/xxxxxx
+  const disabledActions = Boolean(capabilities[SECURITY_FEATURE_ID].configurations);
```

Comment:
Consider:

```typescript
  const disabledActions =
    showAnonymized || Boolean(capabilities[SECURITY_FEATURE_ID].configurations);
```

to ensure the actions are disabled when `showAnonymized` is `true`

Reply:
Actually @andrew-goldstein you meant `showAnonymized` && ...` instead of `showAnonymized` || ...` correct?

Reply:
changed and unit tests added!